rpm/src/main/java/org/eclipse/packager/rpm/VerifyFlags.java

```diff
@@ -0,0 +1,39 @@
+package org.eclipse.packager.rpm;
+
+/**
+ * Constants to identify the RPM verification flags.
+ * See https://github.com/ctron/rpm-builder/issues/41
+ * The name of this enum is questionable.
+ * It should rather be "VerifyFlag" (singular), but I leave it this way, cf. {@link FileFlags}.
+ * The constants and their value are from http://ftp.rpm.org/api/4.14.0/group__rpmvf.html
+ * @since 0.15.2
+ * @author Oliver Matz
+ */
+public enum VerifyFlags
+{
+   MD5(   1 << 0), // '5'
+   SIZE(  1 << 1), // 'S'
+   LINKTO(1 << 2), // 'L'
+   USER(  1 << 3), // 'U'
+   GROUP( 1 << 4), // 'G'
+   MTIME( 1 << 5), // 'T'
+   MODE(  1 << 6), // 'M'
+   RDEV(  1 << 7), // 'D'
+   CAPS(  1 << 8); // 'P'
+   //  VERIFY_CONTEXTS = (1 << 15),  VERIFY_FILES = (1 << 16),  VERIFY_DEPS = (1 << 17),
```

Comment:
Is there a reason for commenting the others out?

Reply:
I think is makes sense to add them all. In the end, this part is only passing them through. So not much processing is done here. For the Maven plugin, it may be a different story.

Reply:
O.K., I will comment in those ones and then push